### ISSUE TITLE
write an ekeyword analog (pkeyword?)

### ISSUE THREAD

**Issue by radhermit** [pkgcore/pkgcore#72]
It would be nice to have a tool that not only could set keywords on ebuilds, but also check the depchains and provide lists of deps also requiring keywording or even just keyword the entire dep chain when requested.

**Comment by vapier** [comment]
the latest ekeyword is written in python fwiw, and includes tests:
https://gitweb.gentoo.org/proj/gentoolkit.git/tree/src/ekeyword?h=gentoolkit-dev

**Comment by radhermit** [comment]
Yep, I might use that for ideas/inspiration. Any reason why it appears to not be installed by default yet for gentoolkit-dev?

**Comment by vapier** [comment]
it's been two years since the last gentoolkit-dev release :x

**Comment by radhermit** [comment]
Hah, well all the more reason for me to write a pkgcore equivalent then and have much shorter release cycles from now on.
